utils/manifests/utils.go

```diff
@@ -156,70 +122,6 @@ func RemoveHelmTemplatingFromCRD(crdyaml *string) {
 	*crdyaml = strings.Join(yamlArr, "\n---\n")
 }
 
-func getCrdnames(s string) []string {
-	s = strings.ReplaceAll(s, "\"", "")
-	s = strings.ReplaceAll(s, " ", "")
-	s = strings.ReplaceAll(s, ",", "")
-	crds := strings.Split(s, "\n")
-	if len(crds) <= 2 {
-		return []string{}
-	}
-	return crds[1 : len(crds)-2] // first and last characters are "[" and "]" respectively
-}
-
-func filterYaml(ctx context.Context, yamlPath string, filter []string, binPath string, inputFormat string) error {
-	var (
-		out bytes.Buffer
-		er  bytes.Buffer
-	)
-	filter = append(filter, "-o", "yaml")
-	getCrdsCmdArgs := append([]string{"--location", yamlPath, "-t", inputFormat, "--filter"}, filter...)
-	cmd := exec.CommandContext(ctx, binPath, getCrdsCmdArgs...)
-	//emptying buffers
-	out.Reset()
-	er.Reset()
-	cmd.Stdout = &out
-	cmd.Stderr = &er
-	err := cmd.Run()
-	if err != nil {
-		return err
-	}
-	path := filepath.Join(os.TempDir(), "/test.yaml")
-	err = populateTempyaml(out.String(), path)
-	if err != nil {
-		return ErrPopulatingYaml(err)
-	}
-	return nil
-}
-
-// cleanup
-func deleteFile(path string) error {
-	err := os.Remove(path)
-	if err != nil {
-		return err
-	}
-	return nil
-}
-
-//TODO: After finalizing on where to keep dictionary.json, use that file to initialize dict map in init()
```

Comment:
I think we still have this as a TODO. If we're deleting it as a comment, do you mind opening up an issue with the same information in order to not lose track of this to-do item?

Reply:
#391

Reply:
Perfectó